frontend/www/js/omegaup/common/runs_chart.js

```diff
@@ -3,7 +3,15 @@ import { API, UI, OmegaUp, T } from '../omegaup.js';
 import Vue from 'vue';
 
 OmegaUp.on('ready', function() {
-  let commonRunsChart = new Vue({
+  const payload = JSON.parse(
+    document.getElementById('runs-chart-payload').innerText,
+  );
+
+  if (payload.total.length === 0) return;
+
+  const minY = payload.total[0] - payload.total[0] / 2.0;
```

Comment:
nit:

```suggestion
  const minY = payload.total.length === 0 ? 0 : payload.total[0] / 2.0;
```

así puedes quitar L10 para que dibuje _algo_ aunque no haya datos.

Reply:
Listo